### ISSUE TITLE
gRPC streams are closed :  [DEBUG] grpc: cleaning up connection to host

### ISSUE THREAD

**Issue by cmvoicu** [fabiolb/fabio#770]
Hi,

I'm trying to use fabio as a proxy for gRPC, but there seems to be an issue with the rpc calls that return a stream. In this scenario the stream is not closed by either client or server but by fabio after some data is received or some idle time:

2020/05/14 14:34:49 [DEBUG] grpc: cleaning up connection to ......

**Comment by cmvoicu** [comment]
This seems to happen only starting with version 1.15.12 (1.5.11 seem to work).
The reason maybe because i don't see in the log file the route add even if in the UI are there:
+ route add rtp-uat-position-service /business.RTPoseService grpcs://rtpuatap002:7772 opts "tlsskipverify=true"
+ route add rtp-uat-position-service /business.RTPoseService grpc://rtpuatap001:7771

**Comment by ishworg** [comment]
What you are seeing is from:
```
			if !hasTarget(target, table) {
				log.Println("[DEBUG] grpc: cleaning up connection to", target.URL.Host)
				cs.Close()
				delete(p.connections, target)
			}
```
afaik, it is normal to see the cleanups happening.

**Comment by cmvoicu** [comment]
Yes, i saw the code, but the gRPC stream it's still open when this happens...so i guess the connection doesn t have a target... it s strange that the same config works for an older version(streams are not closed by fabio)
